README.md

```diff
@@ -49,6 +49,7 @@ If you are interested in GraphQL in general, check out [awesome-graphql](https:/
 | API | Description | Graph*i*QL | Docs/Repo
 | --- | ----------- | :--------: | :-: |
 | Câmara dos deputados Brasil | "GraphQL api to grab the data from the brazilian deputies chamber" | [Try it!](https://graphql-camara-deputados.herokuapp.com/) | [Repo](https://github.com/matheusrocha89/graphql-camara-deputados)
+| Ghibliql | "GhibliQL is a GraphQL wrapper to the Studio Ghibli REST API" | [Try it!](https://ghibliql.herokuapp.com/) | [Repo](https://github.com/kisscool-fr/ghibliql)
```

Comment:
@pseuyi Thanks for the PR 👍 
`Try it` doesn't link to GraphiQL, can you please provide a link to GraphiQL?
Not a critical one so will merge as-is.